meson.build

```diff
@@ -46,28 +46,34 @@ foreach cflag: test_cflags
   endif
 endforeach
 
+with_docs = get_option('with_docs')
```

Comment:
Totally optional: given the way things lay out now, it would make more sense to put this line right before the `if with_docs` block.

Reply:
Done